src/ui/page/dataverse/dataverse.tsx

```diff
@@ -53,133 +53,152 @@ const dataverseItems: DataverseItemDetails[] = [
     type: 'dataspace',
     label: 'Rhizome',
     description:
-      'Rhizome is a data space operated by OKP4, currently under development based on OKP4 technology. Rhizome demonstrates the power of data processing and sharing, and the value we can achieve by effectively connecting different sources of open access agricultural data in different data formats. Rhizome aims to connect as much data as possible and provide valuable visuals and metrics in various agriculture-related areas, such as land use and land management, crop and livestock management, and forest resources and timber industry.'
+      'Rhizome is a data space operated by OKP4, currently under development based on OKP4 technology. Rhizome demonstrates the power of data processing and sharing, and the value we can achieve by effectively connecting different sources of open access agricultural data in different data formats. Rhizome aims to connect as much data as possible and provide valuable visuals and metrics in various agriculture-related areas, such as land use and land management, crop and livestock management, and forest resources and timber industry.',
+    tags: ['Agriculture', 'Open data', 'Dataviz']
```

Comment:
@GeoffreyPicard 
`tags` property should not be carried by the dataversePage because the dataverse entity will only describes the `dataverse` page, not the details one.
There will not have props between components, but we will make a call to anthology to retrieve the data space metadata.

Reply:
Where do the tags go then?

Reply:
Tags should be part of the data space details page, that will handle a different entity from dataverse's one